packages/runtime/lifecycle/test/jest.e2e.config.mjs

```diff
@@ -23,11 +21,11 @@ const config = {
   testEnvironment: "node",
   displayName: "Runtime Lifecycle e2e Test" ,
   extensionsToTreatAsEsm: ['.ts'],
-  testRegex: ".*e2e.ts$",
-  modulePaths: [relative('.')],
+  testRegex: ".*e2e.spec.ts$",
```

Comment:
did this fix the TS-Jest not being in sync?

Reply:
it has fixed nothing ... I think we should get rid of jest for good

Reply:
Will ask what are the cool kids using instead of Jest for TS and ESM